Documentation/ceph-common-issues.md

```diff
@@ -424,6 +432,19 @@ rook-ceph-osd-prepare-minikube-9twvk   0/2     Completed   0          30m
 
 See the section on [why OSDs are not getting created](#osd-pods-are-not-created-on-my-devices) to investigate the logs.
 
+#### CSI Driver
+
+The CSI driver may not be responding to the requests. Look in the logs of the CSI provisioner pod to see if there are any errors
+during the provisioning.
+
+```
```

Comment:
What about the following? Instead of everything in the console section.

There are two provisioner pods:

```console
kubectl -n rook-ceph get pod -l app=csi-rbdplugin-provisioner
```

Get the logs of each of the pods. One of them should be the "leader" and be responding to requests.

```console
kubectl -n rook-ceph logs csi-cephfsplugin-provisioner-d77bb49c6-q9hwq csi-provisioner
```